### DISCUSSION TITLE
Delete persistent-claim

### DISCUSSION THREAD

**Discussion by sreejesh123** [strimzi/strimzi-kafka-operator#5338] [Q&A]
My original setup when created kafka was 

    storage:
      type: jbod
      volumes:
      - id: 0
        type: persistent-claim
        size: 100Gi
        deleteClaim: false
(can I delete the storage manually to reclaim the storage  if   deleteClaim: false

Or I redeploy  kafka/ZK to -  below

    storage:
      type: jbod
      volumes:
      - id: 0
        type: persistent-claim
        size: 100Gi
        deleteClaim: true

then I delete operator - will it delete the storage as well?  

Second Q:  if I have to use SSD as storage type, is the only change needed is   (https://strimzi.io/docs/operators/latest/using.html)

    storage:
        type: persistent-claim
        size: 100Gi
        deleteClaim: true
        selector:
          hdd-type: ssd

should I create kind: storageclass for this to work?

**Comment by scholzj** [comment]
The `deleteClaim` can be changed even for deployed Kafka cluster. But you should be careful about it in production to not loose data after some mistake etc. Deleting the PVC manually after you deleted the cluster is an option as well of course.

As for the second part, I never used the selectors, I normally use StorageClass always with dynamically created storage. I guess the selector makes sense if the storage is not created dynamically?

**Reply by sreejesh123** [reply]
thank you, Its on Dev, trying out few things.  Will make sure the point you raised on data deletion in PROD ,   Second point I will try with both class and selector and get back what works and not :-)
